lib/Dialect/FIRRTL/Transforms/CreateSiFiveMetadata.cpp

```diff
@@ -184,13 +184,32 @@ struct ObjectModelIR {
         module.getLoc(), blackBoxMetadataClass.getBodyBlock());
     auto modEntry = builderOM.create<StringConstantOp>(module.getDefnameAttr());
     auto inDutAttr = builderOM.create<BoolConstantOp>(inDut);
+
+    // Create list of blackbox resources from the provided list
+    SmallVector<Value> libValues;
+    for (StringRef libName : libraries) {
+      Value libNameAttr;
+      if (libName == defName) {
+        libNameAttr = modEntry;
+      } else {
```

Comment:
nit: I think the `else` case is simple enough that parentheses could be elided by our [rule of thumb](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements).